src/main/java/uk/gov/companieshouse/api/accounts/CompanyAccountsApplication.java

```diff
@@ -102,6 +102,7 @@ public void addInterceptors(final InterceptorRegistry registry) {
 
         registry.addInterceptor(smallFullInterceptor)
             .addPathPatterns(
-                "/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full**");
+                "/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full**",
+                "/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full/**");
```

Comment:
We have had similar issues with the double star not matching everything to the end of the url before hence the duplication with `small-full**` and `small-full/**` being the only slight difference here.
It would be good to get this tested once and for all here and documented/commented inline so that we are consistent and happy with it going forwards

Also, while doing this we should ensure that the patterns on interceptors in this block are using the same approach i.e. the extra path with the slash included or not across the board

Reply:
Agreed 👍  It would be helpful to know whether this is necessary, and if so why.

Reply:
After testing this URL `/small-full/current-period` only this pattern `small-full/**` calls the interceptor not `small-full**`

Reply:
In that case, can we remove the `**` from the first pattern? That way the first pattern would cater for the exact `small-full` URI, while the second would cater for any sub resources.

Reply:
Done